jcommune-plugins/questions-n-answers-plugin/src/main/resources/org/jtalks/jcommune/plugin/questionsandanswers/template/questionForm.vm

```diff
@@ -19,7 +19,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 <head>
   <meta name="description" content="${esc.html(${topicDto.topic.branch.name})}">
   <title>
-    #if(${topicDto.topic.branch.name})
+    #if(!${result.hasErrors()})
```

Comment:
Why do we show name of branch in title only if validation errors happened? In this case if user will enter invalid data title of the page will be changed. I don't think that it is expected.